shared/utils/getCurrency.ts

```
@@ -8,6 +8,8 @@ export const getCurrencyFromManager = (
   const currencyResult =
     currency.type === Types.RequestLogic.CURRENCY.ETH
       ? currencyManager.getNativeCurrency(currency.type, currency.network!)
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)